servicemonitor-prometheus-rw-op-system-ns.yaml

```diff
@@ -0,0 +1,106 @@
+# compare k -n risingwave-operator-system get servicemonitor default -o yaml
```

Comment:
Where do we want to add these settings?

Reply:
In a deployment where the Prometheus operator is also installed. This should be an optional setting. We can add a doc about how to apply this. We can leave it to another PR.